JSDemos/Demos/TreeList/ColumnFixing/description.md

```diff
@@ -1 +1,5 @@
-If&nbsp;the TreeList contains many columns, they may exceed the width of&nbsp;the component and cause horizontal scrolling. At&nbsp;the same time, some pivotal columns should be&nbsp;on&nbsp;screen no&nbsp;matter how far the TreeList is&nbsp;scrolled. Such columns can be&nbsp;fixed to&nbsp;the right or&nbsp;to&nbsp;the left of&nbsp;the TreeList using the context menu appearing when a&nbsp;user right-clicks a&nbsp;column header.
\ No newline at end of file
+A horizontal scrollbar appears when the sum width of all columns exceeds the TreeList component's width. The component allows users to fix individual columns at the right or left border so they would remain always visible. To enable this feature, set the **columnFixing**.[enabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columnFixing/#enabled) property to **true**.
+
+To fix or unfix a column, users should right-click the column's header and select the corresponding operation in the context menu. To forbid users to fix or unfix a column, set its [allowFixing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#allowFixing) property to **false**.
```

Comment:
```suggestion
To fix or unfix a column, users should right-click the column's header and select the corresponding operation in the context menu. To prevent users from fixing or unfixing a column, set its [allowFixing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/columns/#allowFixing) property to **false**.
```